src/main/drivers/vtx_rtc6705.c

```diff
@@ -113,13 +113,13 @@ bool rtc6705IOInit(const vtxIOConfig_t *vtxIOConfig)
     // RTC6705 when using SOFT SPI driver doesn't use an SPI device, so don't attempt to initialise an spiInstance.
     SPI_TypeDef *spiInstance = spiInstanceByDevice(SPI_CFG_TO_DEV(vtxIOConfig->spiDevice));
     if (spiInstance && spiSetBusInstance(dev, vtxIOConfig->spiDevice)) {
```

Comment:
dev is not set here yet, use instance:
```suggestion
    if (spiInstance && spiSetBusInstance(&devInstance, vtxIOConfig->spiDevice)) {
        dev = &devInstance;
```

Reply:
HI.I have updated the suggestion, please help me check it.